packages/lwt/lwt.2.5.1/opam

```diff
@@ -40,5 +40,6 @@ depopts: [
 conflicts: [
  "react" {<"1.0.0"}
  "ssl" {< "0.5.0"}
+ "camlp4" {= "4.02+7"} (* lacks "camlp4.quotations.o" *)
```

Comment:
hmm, could you elaborate on this?  I have `lwt-2.5.1` and `camlp4-4.02+7` installed here (including `lwt.syntax`)... what is the problem here? (where is `camlp4.quotations.o` used?)